src/scripts/OSFramework/OSUI/Pattern/InlineSvg/InlineSvg.ts

```diff
@@ -70,11 +70,14 @@ namespace OSFramework.OSUI.Patterns.InlineSvg {
 			console.log(GlobalEnum.WarningMessages.MethodNotImplemented);
```

Comment:
@joselrio Weren't these the console logs we talked about earlier?

Reply:
This consoles must be there since the method is implementation mandatory, it was the call of this methods that have been removed.